### ISSUE TITLE
Incomming messages are ONLY displayed in the overview

### ISSUE THREAD

**Issue by yo-k** [nanu-c/axolotl#89]
Just for the record. There was a bad workaround (->bottom of this ~~ticket~~ comment).
### Description
Incoming messages are only displayed in the overview. In the overview you can see the first part of the last message per contact. If you click on the contact to read the entire message, it is not there. If you restart the signal app, the message is also no longer visible in the overview.

If several messages come from a contact, only the last message per contact is displayed in the overview.

Sometimes, not always, the messages are marked as read on sender side.

Same behavior on groups.

### Steps to Reproduce and actual behavior

1. receive a message from contact
2. open signal app, if not already opened
3. overview page should display preview of message
4. click on that contact
5. no new message
6. ask contact, if message is marked as read (some times it was)

**Expected behavior:**
1. receive message
2. open signal app, if not already opened
3. overview page should display preview of message
4. click on that contact
5. full new message is displayed
6. ask contact, if message is marked as read (should be)

#### Versions
0.7.0

#### Device
bq e4.5 ota10

#### Workaround
1. deinstall
2. reboot
3. install
4. reboot

**Comment by yo-k** [comment]
Would be nice to have a axolotl sparring partner. My family don't like all the test messages :-)
How to remove all the useless messages in axolotl 0.7.0 ?

**Comment by yo-k** [comment]
This issue appeared after traveling by train for a long time. Could be related to #55

**Comment by nanu-c** [comment]
longpress on the on the chatList, I have a simcard only for testing, I will send you the number and you can test what ever you want

:)

**Comment by yo-k** [comment]
with a long press you could select and then remove the contact with the whole thread with all the messages in it. But I want to remove just a few messages in that thread. A long press on a single message does not do anything.

**Comment by nanu-c** [comment]
That's right, we can add this, it's easy

**Comment by yo-k** [comment]
Display of messages seems to be a bit tricky. I have just got a message from a new/unknown number/contact. It was displayed in "Notifications" (Benachrichtigungen). I have clicked on it, with that switching to signal app. No new messages. No new contact. Notifications are empty, cause i clicked on the notification. The message just go away like yesterday. Switched back to signal, opened another contact, closed it, opened another contact, closed it....and then the new message and new contact appeared!

It seems like I could send a message back but I am not sure, yet.
